packages/oid4vci-demo-frontend/src/configs/triall.json

```diff
@@ -1,7 +1,7 @@
 {
   "general": {
     "credentialName": "Triall Guest",
-    "issueCredentialType": "triall2023",
+    "issueCredentialType": "GuestCredential",
```

Comment:
why we're issuing GuestCredential all of a sudden?

Reply:
Huh that should be the type to begin with right. This is the Type that ends up in the credential and that should by GuesCredential for multiple envs like FMA, Triall, DBC

Reply:
This is correct, I'm just not sure why it's defined in two different places

Reply:
Reverted